src/main/java/org/icgc_argo/workflow/search/model/graphql/Analysis.java

```diff
@@ -41,15 +37,8 @@
 
   private String analysisId;
 
-  private AsyncDataFetcher<List<Run>> inputForRunsFetcher;
-
   @SneakyThrows
   public static Analysis parse(@NonNull Map<String, Object> sourceMap) {
     return MAPPER.convertValue(sourceMap, Analysis.class);
   }
-
-  @SneakyThrows
-  public List<Run> getInputForRuns(DataFetchingEnvironment env) {
-    return inputForRunsFetcher.get(env).get();
-  }
```

Comment:
This is a bug. The `get()` doesn't resolve the way I thought and the app never returns any results when quering `analysis.inputForRuns`. I've moved the logic for getting the `inputForRuns` into the runs data fetcher and written it into the build wiring (similar to how `Task.run` is done).